src/content/operations/cleanup/_index.en.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Uninstalling Submariner"
+date: 2020-12-23T21:25:11+01:00
+weight: 20
+---
+
+To properly uninstall Submariner from a cluster, follow the steps below.
+
+1. Delete Submariner related namespaces:
+
+   On data cluster,
```

Comment:
Is "Data cluster" the common name ? 
Shouldn't we say - on each cluster member (or other name we already use in Docs) ?

Reply:
data cluster is common IMO. And broker clusters. Thats the name we use, no?

Reply:
@nyechiel ^